collect_app/src/main/res/values/strings.xml

```diff
@@ -743,4 +743,5 @@
 
     <string name="install_id">Install ID - will replace Device ID in August 2020</string>
     <string name="preference_not_available">Not available</string>
+    <string name="form_instance_file_creation_error">Form instance file could not be created!</string>
```

Comment:
Would you be open to using `loading_form_failed`: "An error occurred while loading the form. Please try again." That way we don't need to introduce a new string for a case we know is rare. I don't think that knowing the failure was about form instance file creation is actionable for users.

Reply:
Do you disagree that having a specific error might help us if someone posts on the forum? Totally hear you on it not being actionable but neither really is a generic "try again" if it's a device/form specific issue.

Reply:
I think it’s very unlikely a user would post. If one does, we can usually track down their crashlytics trace from their device info. So I do prefer something generic.